lib/Models/PWC/Champions.pm

```diff
@@ -0,0 +1,23 @@
+package PWC::Schema::Result::Champion;
```

Comment:
Package name should match folder path. 

I suggest name your package like:

      package Models::PWC::Schema::Result::Champion;

and it should be saved in the folder:
     
      lib/Models/PWC/Schema/Result/Champion.pm

Reply:
@manwar , please should I 
do this for all the tables?